docs/user/sbt.rst

```diff
@@ -39,8 +39,6 @@ Sbt settings
 ======================== =============== =======================================
 Name                     Type            Description
 ======================== =============== =======================================
-``nativeClang``          ``File``        Path to ``clang`` command
-``nativeClangPP``        ``File``        Path to ``clang++`` command
 ``nativeCompileOptions`` ``Seq[String]`` Extra options passed to clang verbatim
```

Comment:
1. We forgot to put ``nativeLinkingOptions`` in there. Can you please modify it to:

```
 ``nativeCompileOptions`` ``Seq[String]`` Extra options passed to clang verbatim during compilation
 ``nativeLinkingOptions`` ``Seq[String]`` Extra options passed to clang verbatim during linking
```

2. Both keys mentioned above should also become tasks as they also discover things from the environment. 

3. Given that ``nativeMode`` is the last setting in there now, lets make everything a task for simplicity.